synnax/pkg/distribution/ontology/search/search.go

```diff
@@ -130,36 +130,50 @@ type Request struct {
 	Type schema.Type
 }
 
-func (s *Index) Search(ctx context.Context, req Request) ([]schema.ID, error) {
-	ctx, span := s.T.Prod(ctx, "search")
-	words := strings.FieldsFunc(req.Term, func(r rune) bool { return r == ' ' || r == '_' || r == '-' })
-	q := bleve.NewConjunctionQuery(lo.Map(words, func(word string, _ int) query.Query {
-		fuzzyQ := bleve.NewMatchQuery(word)
-		fuzzyQ.SetFuzziness(1)
-		regexQ := bleve.NewRegexpQuery(".*[_\\.-]" + word + ".*")
-		prefixQ := bleve.NewPrefixQuery(word)
-		exactQ := bleve.NewMatchQuery(word)
-		exactQ.SetFuzziness(0)
-		exactQ.SetBoost(100)
-		return bleve.NewDisjunctionQuery(exactQ, prefixQ, regexQ, fuzzyQ)
-	})...)
+func assembleWordQuery(word string, _ int) query.Query {
+	fuzzyQ := bleve.NewMatchQuery(word)
+	fuzzyQ.SetFuzziness(1)
```

Comment:
if you're gonna hard code the number, can you put a comment here indicating what the number represents or the scale